src/components/cars/addcars.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import logo from '../../images/cars-bnb-logo.png';
+import '../../styles/addcars.css';
+
+function Addcars() {
+  return (
+    <div className="big">
+      <div className="container">
+        <form>
+          <div className="brand-logo">
+            {' '}
+            <img src={logo} alt="Cars bnb logo" />
+            {' '}
+          </div>
+
+          <div className="inputs">
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label className="green">Model</label>
+            <input className="input2" type="text" placeholder="Model" />
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label className="green">Brand</label>
+            <input className="input2" type="text" placeholder="Brand" />
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label className="green" htmlFor="date">
+              date
+            </label>
+
+            <input
+              className="input2"
+              type="date"
+              id="date"
+              name="trip-date"
+              min="1990-01-01"
+              max="2022-12-31"
+            />
```

Comment:
Awesome work done on this add car form. 

But how do we validate the data supplied by the user? My suggestion is that we use [React-hook-form](https://react-hook-form.com/get-started); 
Great work bro.